packages/elements-core/src/components/TryIt/Body/__tests__/FormDataBody.test.tsx

```diff
@@ -0,0 +1,102 @@
+import '@testing-library/jest-dom';
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { FormDataBody, FormDataBodyProps } from '../FormDataBody';
+
+describe('FormDataBody', () => {
+  it('when the top level of the schema is oneOf', () => {
+    // Arrange
+    const props: FormDataBodyProps = {
+      mediaTypeContent: {
+        id: '493afac014fa8',
+        mediaType: 'application/x-www-form-urlencoded',
+        encodings: [],
+        schema: {
+          oneOf: [
+            {
+              title: 'Authenticate',
+              description: 'supply username and password for refresh and access tokens',
+              type: 'object',
+              properties: {
+                username: {
+                  type: 'string',
+                },
+                password: {
+                  type: 'string',
+                },
+              },
+            },
+            {
+              title: 'Access Token Refresh',
+              description: 'supply a refresh token for new refresh and access tokens',
+              type: 'object',
+              properties: {
+                refresh_token: {
+                  type: 'string',
+                },
+              },
+            },
+          ],
+          // 'x-stoplight': {
+          //   id: 'fy4lgsesqt84g',
+          // },
+          // 'x-internal': false,
```

Comment:
Can this be removed?
```suggestion
```

Reply:
Thanks for catching that.  I'll get rid of 'em.

Reply:
I removed both in ac00aab1.